libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -124,10 +124,26 @@ singleton AP_Notify alias notify
 singleton AP_Notify method play_tune void string
 singleton AP_Notify method handle_rgb void uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
 
+include AP_Proximity/AP_Proximity.h
+
+singleton AP_Proximity alias proximity
+singleton AP_Proximity method num_sensors uint8_t
+singleton AP_Proximity method get_object_count uint8_t
+singleton AP_Proximity method get_closest_object boolean float'Null float'Null
+singleton AP_Proximity method get_object_angle_and_distance boolean uint8_t 0 UINT8_MAX float'Null float'Null
+
 include AP_RangeFinder/AP_RangeFinder.h
 
 singleton RangeFinder alias rangefinder
 singleton RangeFinder method num_sensors uint8_t
+singleton RangeFinder method has_orientation boolean Rotation'enum ROTATION_NONE ROTATION_ROLL_90_PITCH_315
```

Comment:
```suggestion
singleton RangeFinder method has_orientation boolean Rotation'enum ROTATION_NONE ROTATION_MAX-1
```

I'd change all of these to use `ROTATION_MAX-1` as their upper limits, that way we just cope with new rotations as they get added to the enum for whatever reason.

Reply:
I suggested to purposefully excluding the pitch 7 rotation, doesn't really make any sense for a rangefinder. There is also rotation custom too of course....

Reply:
I'd include it, just because we might want ones afterwards. And it's not obvious from an API level why we'd be unhappy with any valid orientation you could have set. Custom is obviously a problem (and not supported for the rangefinder library), but if it's in the enum I can't see a reason to cut it out.

Reply:
I agree that there may be problems in the future.